training/src/main/scala/com/airbnb/aerosolve/training/SplineTrainer.scala

```diff
@@ -352,7 +510,7 @@ object SplineTrainer {
                   dropout : Double,
                   margin : Double) : Double = {
     val flatFeatures = Util.flattenFeatureWithDropout(fv, dropout)
-    val prediction = model.scoreFlatFeatures(flatFeatures)
+    val prediction = model.scoreFlatFeatures(flatFeatures) / (1.0 - dropout)
```

Comment:
Double checking http://www.cs.toronto.edu/~rsalakhu/papers/srivastava14a.pdf
"In this paper, we described dropout as a method where we retain units with probability p
at training time and scale down the weights by multiplying them by a factor of p at test time.
Another way to achieve the same effect is to scale up the retained activations by multiplying
by 1/p at training time and not modifying the weights at test time."
I'm doing it by the 2nd way so adding a comment